workflows/epigenetics/ChIPseq_PE/ChIPseq_PE-tests.yml

```diff
@@ -0,0 +1,106 @@
+- doc: Test outline for ChIPseq_PE
+  job:
+    PE fastq input:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: Collection
+        type: paired
+        identifier: wt_H3K4me3
+        elements:
+        - identifier: forward
+          class: File
+          location: https://zenodo.org/record/1324070/files/wt_H3K4me3_read1.fastq.gz
+          filetype: fastqsanger
+        - identifier: reverse
+          class: File
+          location: https://zenodo.org/record/1324070/files/wt_H3K4me3_read2.fastq.gz
+          filetype: fastqsanger
+    adapter_forward: 'GATCGGAAGAGCACACGTCTGAACTCCAGTCAC'
+    adapter_reverse: 'GATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT'
+    reference_genome: 'mm10'
+    effective_genome_size: 1870000000
+  outputs:
+    MultiQC webpage:
+      asserts:
+        - that: "has_text"
+          text: "wt_H3K4me3"
+        - that: "has_text"
+          text: "<a href="#cutadapt_filtered_reads" class="nav-l2">Filtered Reads</a>"
+        - that: "has_text"
+          text: "<a href="#bowtie2" class="nav-l1">Bowtie 2 / HiSAT2</a>"
+    'MultiQC on input dataset(s): Stats':
+      element_tests:
+        bowtie2:
+          asserts:
+            has_text: 
+              text: "46307\t46307\t1292"
+        cutadapt:
+          asserts:
+            has_text: 
+              text: "4.0\t50000\t587\t550\t3693\t46307\t5100000"
+        general_stats:
+          asserts:
+            has_text: 
+              text: "201.0\t0.0\t11\t98.68"
+        macs:
+          asserts:
+            has_text:
+              text: "11\t201.0\t41724.0\t41724.0\t1.0"
+        sources:
+          asserts:
+            has_n_lines: 
+              n: 3
+    filtered BAM:
+      element_tests:
+        wt_H3K4me3:
+          asserts:
+            has_size:
+              value: 5008461
+              delta: 200000
+    MACS2 summits:
+      element_tests:
+        wt_H3K4me3:
+          has_n_lines: 11
```

Comment:
Oups I think this is not working...

Reply:
May be explaining why I got an error at MACS2 step ?

Reply:
You got an error in planemo test?

Reply:
No, this is happening in running workflow on usegalaxy.org

Reply:
This is not related then because this is just the test to run in the continuous integration on CI...
Can you share your history with me: lucille.delisle@epfl.ch

Reply:
I launch another run. I had an error with my genome size. I hope this is the reason why it failed otherwise I will share the history